### PR TITLE
OpenConceptLab/ocl_issues#965 | using whitenoise to serve staticfiles for non-dev mode (with gunicorn)

### PR DESCRIPTION
**Pull request by snyaggarwal**

### CODE REVIEW

File: core/urls.py [hunk 1]
```diff
@@ -51,7 +50,6 @@
     url(r'^swagger/$', SchemaView.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
     url(r'^redoc/$', SchemaView.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
     path('healthcheck/', include('core.common.healthcheck.urls')),
-    path('admin/', admin.site.urls, name='admin_urls'),
```
Comment (rkorytkowski) [inline core/urls.py:54]: Why are you removing this?

File: core/settings.py [hunk 2]
```diff
@@ -266,6 +265,9 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+STATIC_ROOT = os.path.join(PROJECT_DIR, 'static')
```
Comment (rkorytkowski) [inline core/settings.py:269]: No need to call collectstatic in Dockerfile?

File: core/urls.py (hunk 1 above)
Comment (snyaggarwal) [reply core/urls.py:54]: we don't use any admin routes/features. And also, with whitenoise all the calls were getting redirected to Django Login.

File: core/settings.py (hunk 2 above)
Comment (snyaggarwal) [reply core/settings.py:269]: ya collectstatic wasn't needed.

**Comment by rkorytkowski** [comment]
Thanks @snyaggarwal. Committed applying a few of your ideas at https://github.com/OpenConceptLab/oclapi2/commit/2f6df521d077e7887b094e1da403eb76d56ced8a

**Comment by snyaggarwal** [comment]
Closing this pull request as this has been added on master already.

**Pull request closed without merging**